docs/root/configuration/overview/extension.rst

```diff
@@ -69,6 +69,8 @@ follows:
 Discovery service
 ^^^^^^^^^^^^^^^^^
 
+HTTP Filters
```

Comment:
Might consider underlying with " (see https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings)

Reply:
After reordering the texts, I think it's better to just remove "HTTP filters" and "Listener Filters" because those are already in the starting text.